pkg/localstorev2/internal/reserve/reserve.go

```diff
@@ -93,7 +99,15 @@ func New(
 }
 
 // Put stores a new chunk in the reserve and returns if the reserve size should increase.
-func (r *Reserve) Put(ctx context.Context, store internal.Storage, chunk swarm.Chunk) (bool, error) {
+func (r *Reserve) Put(ctx context.Context, store internal.Storage, chunk swarm.Chunk) (inc bool, err error) {
+	defer func(start time.Time) {
+		r.metrics.MethodCallsDuration.WithLabelValues("Put").Observe(float64(time.Since(start).Nanoseconds()))
```

Comment:
nitpick: I would recommend using `time.Since(start).Seconds()` as it outputs float64 too, it's easier to  work with when building dashboards